.github/workflows/release.yml

```
@@ -10,6 +10,13 @@ on:
         description: 'Tag to build retroactive release for (e.g. v0.9.0)'
         required: true
         type: string
+permissions:
+  contents: read
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
 
 permissions: {}
```

actionlint (1.7.12)

[error] key "permissions" is duplicated in "workflow" section. previously defined at line:13,col:1

(syntax-check)